ci-operator/step-registry/openshift-qe/write-perfscale-results/openshift-qe-write-perfscale-results-commands.sh

```diff
@@ -42,9 +39,10 @@ ls sandman
 
 source ./sandman/data/workload.sh
 
-pushd ocp-qe-perfscale-ci
+pushd ocp-qe-perfscale-ci || exit 0
 
 pip install -r requirements.txt
-pushd write_to_sheet
+pushd write_to_sheet || exit 0
 
-python prow_write.py
\ No newline at end of file
+python prow_write.py
+exit 0
```

Comment:
Can we may be try handling from prow side or other alternatives instead of always passing? Pls see my comment.